dev-util/bear/bear-3.1.3.ebuild

```diff
@@ -0,0 +1,91 @@
+# Copyright 2020-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..12} )
+
+inherit cmake python-any-r1
+
+DESCRIPTION="Build EAR generates a compilation database for clang tooling"
+HOMEPAGE="https://github.com/rizsotto/Bear"
+SRC_URI="https://github.com/rizsotto/Bear/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="amd64 ~arm64 ~loong ~ppc64 ~riscv x86"
+IUSE="test"
+
+RDEPEND="
+	>=dev-cpp/nlohmann_json-3.11.2:=
+	>=dev-db/sqlite-3.14:=
+	>=dev-libs/libfmt-9.1.0:=
+	dev-libs/protobuf:=
+	>=dev-libs/spdlog-1.11.0:=
+	>=net-libs/grpc-1.49.2:=
+"
+
+DEPEND="${RDEPEND}
+	test? (
+		>=dev-cpp/gtest-1.13
+	)
+"
+
+BDEPEND="
+	virtual/pkgconfig
+	test? (
+		sys-devel/libtool
+		$(python_gen_any_dep '
+			dev-python/lit[${PYTHON_USEDEP}]
+		')
+	)
+"
+
+RESTRICT="!test? ( test )"
+
+S="${WORKDIR}/${P^}"
+
+pkg_setup() {
+	use test && python-any-r1_pkg_setup
+}
+
+src_prepare() {
+	cmake_src_prepare
+	# Turn off testing before installation
+	sed -i 's/TEST_BEFORE_INSTALL/TEST_EXCLUDE_FROM_MAIN/g' CMakeLists.txt || die
+}
+
+src_configure() {
+	local mycmakeargs=(
+		-DENABLE_UNIT_TESTS="$(usex test)"
+		-DENABLE_FUNC_TESTS="$(usex test)"
+	)
+	cmake_src_configure
+}
+
+src_test() {
```

Comment:
this whole src_test feels horrible and like it should be fixed properly but it's ok to do it another time